scaleway/data_source_registry_namespace_beta.go

```diff
@@ -0,0 +1,85 @@
+package scaleway
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/registry/v1"
+)
+
+func dataSourceScalewayRegistryNamespace() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceScalewayRegistryNamespaceRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Can you use the `datasourceSchemaFromResourceSchema` function?
[Example](https://github.com/terraform-providers/terraform-provider-scaleway/blob/master/scaleway/data_source_instance_server.go#L11-L24)